### PR TITLE
Update tracing.md

### PR DESCRIPTION
**Pull request by itsliamdowd**
Update tracing.md

### CODE REVIEW

File: doc/tracing.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ added to and removed (spent) from the cache when we connect a new block.
 `getblocktemplate` call `TestBlockValidity()`, which applies the UTXO set
 changes to a temporary cache. Similarly, mempool consistency checks, which are
 frequent on regtest, also apply the the UTXO set changes to a temporary cache.
-Changes to the _main_ UTXO cache and to temporary caches trigger the tracepoints.
+Changes to the _main_ UTXO cache and temporary caches trigger the tracepoints.
```
Comment (luke-jr) [inline doc/tracing.md:120]: Should this be "or"?

**Comment by 0xB10C** [comment]
otherwise, lgtm

**Comment by fanquake** [comment]
@SkiingIsFun123 are you planning on updating this? Note that you'll need to squash all commits, and you can write a proper commit message, i.e 'doc: commit message explaining change'

**Comment by fanquake** [comment]
Closing up for grabs.

**Pull request closed without merging**